### DISCUSSION TITLE
IBAN already in use

### DISCUSSION THREAD

**Discussion by UltraBlackLinux** [firefly-iii/firefly-iii#9003] [Firefly III]
Hey there,
to have a better time with the importer I wanted to add an IBAN to each one of my accounts. I'm not quite sure if that was me clicking on the wrong account, but when I tried to edit my main account, it somehow edited the wrong account and added the IBAN there. When I went ahead and tried to fix that, the IBAN I use for my savings account is now somehow used, even though all my other accounts have different IBANs set.
Not even deleting every existing account helped.
Any idea what I'm supposed to do now?
Thanks!

I'm using the docker setup, all on the develop branch because it has a fix for something that I cannot remember what it was. I've already tried updating and restarting the containers, to no avail.

**Comment by JC5** [comment]
Could it be in use by an expense account or a revenue account perhaps?

**Reply by UltraBlackLinux** [reply]
I've checked all the accounts in the accounts view, and have even deleted each and every single one of them, but to no avail.

**Reply by JC5** [reply]
Then I don't know, sorry.
